streams/src/test/java/org/apache/kafka/streams/processor/internals/RecordCollectorTest.java

```diff
@@ -754,7 +755,7 @@ public void shouldForwardFlushToStreamsProducer() {
         final StreamsProducer streamsProducer = mock(StreamsProducer.class);
         when(streamsProducer.eosEnabled()).thenReturn(false);
         doNothing().when(streamsProducer).flush();
-
+        when(streamsProducer.sendException()).thenReturn(new AtomicReference<>(null));
```

Comment:
Needed to add this stub to a couple of tests so that they do not throw a `NullPointerException` because the mock return `null` for `sendException()`.